wazo_calld/plugins/calls/bus_consume.py

```diff
@@ -235,6 +243,15 @@ def _relay_channel_left_bridge(self, event):
                 logger.debug('channel %s not found', participant_channel_id)
                 return
 
+            if channels_in_bridge > 1:
+                # If this is the last channel of the bridge we want to keep the last call direction
```

Comment:
Please reformulate this comment: the condition in the comment is the reverse of the condition in the code, so I had a hard time understanding the comment.

Reply:
I will only remove this comment, I now find it pretty useless.